src/main/java/com/findmyrecycling/fmrenterprise/FindMyRecyclingController.java

```diff
@@ -21,66 +20,56 @@ public class FindMyRecyclingController {
     IFacilityService facilityService;
 
     @RequestMapping("/")
-    public String index(Model model) {
-        RecyclableMaterial recyclableMaterial = new RecyclableMaterial();
-        Facility facility = new Facility();
-        recyclableMaterial.setMaterialName("");
-        facility.setFacilityAddress("");
-        model.addAttribute(recyclableMaterial);
+    public String index() {
```

Comment:
This doesn't need to be here. It's just clutter because it wasn't being used anywhere, really.